python/pyspark/sql/streaming/stateful_processor_api_client.py

```diff
@@ -115,7 +117,13 @@ def get_value_state(self, state_name: str, schema: Union[StructType, str]) -> No
         response_message = self._receive_proto_message()
         status = response_message[0]
         if status != 0:
-            raise PySparkRuntimeError(f"Error initializing value state: " f"{response_message[1]}")
+            raise PySparkRuntimeError(
```

Comment:
Shall we give a better error class as it's user facing error? You can revert back and file a JIRA ticket for this as well to defer the change.

Reply:
I'd expect having dedicated error class, if Scala version of the implementation uses the error class then use the same, otherwise define a new one.